packages/driver/cypress/fixtures/isolated-runner-inner.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Isolated Runner Fixture</title>
+  </head>
+  <body>
+    Some generic content
+
+    <div>
+      <span class="foo">Foo</span>
+      <span class="bar">Bar</span>
+      <span class="baz">Baz</span>
+      <span class="qux">Qux</span>
+    </div>
+
+    <a href="/">link</a>
+
+    <a id="hashchange" href="#hashchange">hashchange</a>
+
+    <a id="dimensions" href="/fixtures/dimensions.html">dimensions</a>
+
+    <div>
+      <button class="sync-error">Sync Error</button>
+      <button class="async-error">Async Error</button>
+    </div>
+
+    <script>
+      document.querySelector(".sync-error").addEventListener('click', function () {
+        qux.bar()
+      })
+
+      document.querySelector(".async-error").addEventListener('click', function () {
+        setTimeout(function () {
+          qax.bar()
```

Comment:
Example of why meaningful english words might be better: qax should be qux, but our editors won't catch it. Additionally, it's difficult to figure out what this method call is supposed to reference in the test.

I know we do this throughout the codebase. Maybe it's a habit worth breaking.